index.html

```diff
@@ -4,9 +4,81 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>Document</title>
+	<title>Marc Raphael Portofolio</title>
+
+	<!--Fonts-->
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+
+	<!--css links-->
+	<link rel="stylesheet" href="./style.css">
 </head>
 <body>
-	
+	<!-- the page's header -->
+	<header class="pg-hdr">
+		<h3>John Doe</h3>
+		<button type="button" class="btn">
+			<img src="./images/buttons/hamburger.svg" alt="menu-pic" >
+		</button>
+	</header>
+
+	<!-- the main content -->
+	<main id="main">
+		<section id="sct1">
+			<div class="titles">
+				<h1>Laura Jong</h1>
+				<h2>Software Developer</h2>
+			</div>
+			<p class="para">
+				I can help you build a product,
+				feature or website Look through some of my work and experience!
+				If you like what you see and have a project you need coded, don’t hestiate to contact me.
+			</p>
+
+			<!-- Soft Skills -->
+			<div class="skills">
+				<ul class="competences cmps-1">
+          <li class="cmp-ttl"><h3>Languages</h3></li>
+          <li class="competence">JavaScript(ES5/ES6)</li>
+          <li class="competence">Ruby</li>
+          <li class="competence">HTML</li>
+          <li class="competence">CSS</li>
+        </ul>
+
+        <ul class="competences cmps-2">
+          <li class="cmp-ttl"><h3>Frameworks</h3></li>
+          <li class="competence">Ruby on rails</li>
+          <li class="competence">Bootstrap</li>
+          <li class="competence">Rspec</li>
+          <li class="competence">Capibara</li>
+          <li class="competence">Selenium</li>
+        </ul>
+
+        <ul class="competences cmps-3">
+          <li class="cmp-ttl"><h3>Skills</h3></li>
+          <li class="competence">Codekit</li>
+          <li class="competence">Codepen</li>
+          <li class="competence">Github</li>
+          <li class="competence">GitLab</li>
+          <li class="competence">Terminal</li>
+        </ul>
+			</div>
+
+			<!-- Socials -->
+			<div class="join">
+				<ul class="Socials">
+					<li><a href="#" class="social"> <img src="./images/buttons/github.svg" alt="Github-pic" /></a></li>
+					<li><a href="#" class="social"> <img src="./images/buttons/linkedin.svg" alt="linkedin-pic" /></a></li>
+					<li><a href="#" class="social"> <img src="./images/buttons/angellist.svg" alt="angellist-pic" /></a></li>
+					<li>  <a href="#" class="social"> <img src="./images/buttons/twitter.svg" alt="twitter-pic" /></a></li>
+					<li><a href="#" class="social"> <img src="./images/buttons/medium.svg" alt="medium-pic" /></a></li>
+				</ul>
+			</div>
+
+			 <!--Transition to work section -->
+			 <button type="button" class="btn-work"> <img src="./images/buttons/down_arrow.svg" alt=""> <span>Work</span> </button>
```

Comment:
- [ ] The buttons should have all design specifications implemented i.e hover, active and pressed.

Reply:
this button is not clickable

Reply:
there's no style for it in the template